gslib/tests/testcase/integration_testcase.py

```diff
@@ -273,6 +276,26 @@ def _SetObjectCustomMetadataAttribute(self, provider, bucket_name,
       self.xml_api.PatchObjectMetadata(bucket_name, object_name, obj_metadata,
                                        provider=provider)
 
+  def get_random_ascii_chars(self, size, seed=None):
+    """Generates random ASCII characters up to a given size.
+
+    Args:
+      size: Integer quantity of characters to generate.
+      seed: A seed may be specified for deterministic behavior.
+            None is used as the default value.
+
+    Returns:
+      String of length equal to size argument.
```

Comment:
In Python 3, this isn't a "string" :) Please differentiate between unicode string and bytes/byte-sequences in your docstrings (maybe by using `six.text_type` and `bytes`/`six.binary_type`?)

Reply:
Yes!! I updated the docstring to clarify explicitly that it's returning an ASCII encoded bytestring, with a note about Py2 / Py3 strings :spiral_notepad: